### PR TITLE
Matter Sensor: Add HEIMAN smart CO sensor fingerprint

### PR DESCRIPTION
**Pull request by ctowns**
Add fingerprint as part of WWST certification for HEIMAN smart CO sensor

### CODE REVIEW

File: drivers/SmartThings/matter-sensor/fingerprints.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@ matterManufacturer:
     vendorId: 0x120B
     productId: 0x1031
     deviceProfileName: contact-battery
+   - id: "4619/4098"
+    deviceLabel: Smart co sensor
```
Comment (ctowns) [inline drivers/SmartThings/matter-sensor/fingerprints.yml:41]: I am following up to see if this name is intentional. Prior HEIMAN devices have included the brand name and have been capitalized in the device label

Comment (hcarter-775) [reply drivers/SmartThings/matter-sensor/fingerprints.yml:41]: has there been any follow-up on this?

Comment (ctowns) [reply drivers/SmartThings/matter-sensor/fingerprints.yml]: Yes I changed it to better match the other device labels

**Review by hcarter-775 — APPROVED** [review]

**Comment by ctowns** [comment]
Updated profile to be `co-comeas` per discussion here: https://smartthings.slack.com/archives/C03QT78EZ0B/p1725565115985549

**Review by nickolas-deboom — APPROVED** [review]

**Pull request merged**